config/heroes/stronghold.json

```diff
@@ -95,8 +95,8 @@
 		"specialty" : {
 			"bonuses" : {
 				"offence" : {
-					"subtype" : "skill.offence",
-					"type" : "SECONDARY_SKILL_PREMY",
+					"subtype" : 0,
+					"type" : "PERCENTAGE_DAMAGE_BOOST",
```

Comment:
Why some heroes got `PERCENT_TO_TARGET_TYPE` type while some (including this one) kept `PERCENT_TO_BASE`?

Reply:
@IvanSavenko I can make it PERCENT_TO_TARGET_TYPE too, but it can break mods which assume than specialities will work over artifacts than boosts SECONDARY_SKILL_PREMY. It breaks such cases in PERCENT_TO_TARGET_TYPE anyway.